packages/python/test/googlestyledocparam.test.tsx

```diff
@@ -0,0 +1,167 @@
+import { d } from "@alloy-js/core/testing";
+import { expect, it } from "vitest";
+import { GoogleStyleDocParam } from "../src/components/GoogleStyleDocParam.jsx";
+import { PyDoc } from "../src/index.js";
+
+it("name only", () => {
+  const template = (
+    <PyDoc>
+      <GoogleStyleDocParam name="somebody" />
+    </PyDoc>
+  );
+
+  expect(template).toRenderTo(
+    d`
+          """
+              somebody
+          """
+        `,
+  );
+});
+
+it("name and type", () => {
+  const template = (
+    <PyDoc>
+      <GoogleStyleDocParam name="somebody" type="str" />
+    </PyDoc>
+  );
+
+  expect(template).toRenderTo(
+    d`
+          """
+              somebody (str)
```

Comment:
This would be rendered inside the Args block, like the param1, param2 and param3 below:

```
def example_function(param1, param2=None, param3: int = 10):
    """
    This is an example function.

    Args:
        param1: The first parameter.
        param2 (:obj:`str`, optional): The second parameter. Defaults to None.
        param3 (int, optional): The third parameter. Defaults to 10.
    """
    pass
```